app/controllers/sessions_controller.rb

```diff
@@ -6,8 +6,13 @@ def new
   def create
     user = User.find_by(email: params[:email])
     if user and user.authenticate(params[:password])
-      session[:user_id] = user.id
-      redirect_to home_path, flash: {success: "You are signed in, enjoy!"}
+      if user.active?
+        session[:user_id] = user.id
+        redirect_to home_path, flash: {success: "You are signed in, enjoy!"}
```

Comment:
Instead of doing this - check [add_flash_types](http://api.rubyonrails.org/classes/ActionController/Flash/ClassMethods.html#method-i-add_flash_types)